data/json/recipes/ammo/other.json

```
@@ -83,19 +83,33 @@
     "subcategory": "CSC_AMMO_OTHER",
     "skill_used": "fabrication",
     "difficulty": 1,
-    "time": "45 m",
+    "time": "30 m",
     "autolearn": true,
     "qualities": [ { "id": "HAMMER", "level": 1 }, { "id": "SAW_M", "level": 1 } ],
     "components": [ [ [ "rebar", 2 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "steel_rail",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_OTHER",
+    "id_suffix": "welding",
+    "skill_used": "fabrication",
+    "difficulty": 6,
+    "time": "15 m",
+    "batch_time_factors": [ 90, 1 ],
+    "autolearn": true,
+    "using": [ [ "welding_standard", 30 ], [ "steel_standard", 1 ] ],
+    "qualities": [ { "id": "CUT", "level": 1 } ]
```

Could potentially be `SAW_M` instead since I assume you're cutting and sharpening the steel with this tool? Is so:
```suggestion
    "qualities": [ { "id": "SAW_M", "level": 1 } ]
```